### PR TITLE
Add Travis CI badge to README

### PR DESCRIPTION
**Pull request by caniszczyk**
https://travis-ci.org/opencontainers/runc

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-[![Build Status](https://jenkins.dockerproject.org/buildStatus/icon?job=runc Master)](https://jenkins.dockerproject.org/job/runc Master)
-
-## runc
+## runc [![Build Status](https://travis-ci.org/opencontainers/runc.svg?branch=master)](https://travis-ci.org/opencontainers/runc) [![Build Status](https://jenkins.dockerproject.org/buildStatus/icon?job=runc Master)](https://jenkins.dockerproject.org/job/runc Master)
```
Comment (dqminh) [inline README.md]: should we just remove the dockerproject link here ? It returns 404 for me.

Comment (caniszczyk) [inline README.md]: @dqminh it's up to the @opencontainers/runc-maintainers IMHO

Comment (cyphar) [inline README.md]: Yeah. those links don't work for me. Let's just include the Travis CI links for now.

Comment (caniszczyk) [inline README.md]: @cyphar @dqminh ok I removed it and just left the Travis CI link only! woooo

**Comment by cyphar** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**